primitives/src/locktime/absolute.rs

```diff
@@ -405,12 +405,25 @@ mod tests {
 
     #[test]
     fn display_and_alternate() {
+        let lock_by_height = LockTime::from_height(741_521).expect("valid height");
+        let lock_by_time = LockTime::from_time(1_653_195_600).expect("May 22nd 2022, 5am UTC");
```

Comment:
Funny idea but generally, we write why we expect it to work - e.g. "the number is above 500000000"

Reply:
I changed them to unwrap and put the date in a comment